lib/defaults/defaults.go

```diff
@@ -1177,7 +1177,11 @@ var BaseTaintsVersion = semver.Must(semver.NewVersion("4.36.0"))
 
 // BaseUpdateVersion sets the minimum version that this binary
 // can update
-var BaseUpdateVersion = semver.Must(semver.NewVersion("3.51.0"))
+var BaseUpdateVersion = semver.Must(semver.NewVersion("5.0.0"))
```

Comment:
Question: I opened this issue a couple weeks ago (https://github.com/gravitational/gravity/issues/536) and it looks like it's already covered by this variable. The question is, how procedurally do we ensure this variable is kept up to date? 

Just thinking out loud, can we expose it in a way that the robotest fails if it's test matrix doesn't test the BaseUpdateVersion? Or some other way to ensure that it's flagged as needing an update?

Reply:
Currently this variable is a baseline of sorts - it does not identify a specific version so even if exposed it would not be tested against. We could instead use a specific version so this became testable as you described.
And there're no provisions to keep it automatically up-to-date as afaic there's no good condition to know when to switch to the new base version other than maybe the version of kubernetes that the bundled runtime application sports.
We should continue discussing this in the issue you [created](https://github.com/gravitational/gravity/issues/536) though.